packages/core/src/browser/fetchObservable.ts

```diff
@@ -113,5 +116,61 @@ function afterSend(
       observable.notify(context)
     }
   }
-  responsePromise.then(monitor(reportFetch), monitor(reportFetch))
+
+  const reportFetchResolve = (response: Response) => {
+    let stopInstruments: Array<() => void> = []
+    let timeOutId: null | number = null
+
+    Promise.race([
+      new Promise((resolve) => {
+        stopInstruments = (['json', 'text', 'formData', 'blob', 'arrayBuffer'] as OverloadMethodKeys[]).map(
+          (method) => {
+            const { stop } = instrumentMethod(
+              response,
+              method,
+              (originalMethod: () => Promise<any>) =>
+                function () {
+                  const res = originalMethod.call(response)
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏ ‏Should not you send `this` and `args` like in [here](https://github.com/DataDog/browser-sdk/blob/1bf03b1dc17d8d3c1aa3d729c44bbd17361c6ab1/packages/core/src/tools/instrumentMethod.ts#L60) to avoid changing the scope and be certain of the parameters?

Reply:
Good point on using `this`, I will update. These methods however do not have `args` so on need